pkg/nuctl/command/deploy.go

```diff
@@ -689,8 +698,25 @@ func (d *deployCommandeer) enrichBuildConfigWithArgs() {
 
 func (d *deployCommandeer) betaDeploy(ctx context.Context, args []string) error {
 
+	if d.redeployFromReportFile {
+		manifest, err := nuctlcommon.NewPatchManifestFromFile(d.reportFilePath)
+		if err != nil {
+			return errors.Wrap(err, "Problem with reading report file")
+		}
+		d.inputManifest = manifest
+		retryableFunctions := d.inputManifest.GetRetryableFunctionNames()
+		d.rootCommandeer.loggerInstance.InfoWith("Redeploying failed functions from report file",
+			"report file", d.reportFilePath,
+			"functions", retryableFunctions)
+		args = append(args, retryableFunctions...)
+	}
+
 	if len(args) == 0 {
 
+		if d.redeployFromReportFile {
+			return errors.New("No retryable functions to redeploy")
```

Comment:
Don't think this is an error, log it and return nil.

Reply:
that's right, fixed